### DISCUSSION TITLE
Preserve blank lines within text in cell

### DISCUSSION THREAD

**Discussion by gnaneshwar441** [jsvine/pdfplumber#452] [Ask for help with specific PDFs]
Hello,

In the example below there are 6 lines (including blank) within a cell in a table

```
Line1
Line2
Line3
Line4
<blank>
Line5
```

After I ran extract_tables() I'm getting all the lines as expected except for the blank line
Output:
```
Line1\n
Line2\n
Line3\n
Line4\n
Line5
```

I was expecting:
```
Line1\n
Line2\n
Line3\n
Line4\n
\n
Line5
```

so that when i split the text within a cell by '\n' I must get `['Line1','Line2','Line3','Line4','','Line5']`

What table settings must I apply to preserve blank lines within a cell?

Thanks

**Comment by samkit-jain** [comment]
Hi @gnaneshwar441  Appreciate your interest in the library. A newline is added when the difference between the `doctop` of one character and the `doctop` of the next character is greater than `y_tolerance` (defaults to 3). It only adds a single new line and does not add multiple new lines. Meaning, if there is say `hello` written in the header and `world` in the footer on a page with nothing in between, the result will be
```
hello
world
```
and not
```
hello


...many new lines...


world
```

If you could share the PDF with any sensitive information redacted, then I can have a look and see if there is a workaround that might be of help to you.
